### DISCUSSION TITLE
TypeError: readonly attribute when compiled with Nuitka using --standalone and --onefile

### DISCUSSION THREAD

**Discussion by xandaaah** [fastapi/typer#832] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the Typer documentation, with the integrated search.
- [X] I already searched in Google "How to X in Typer" and didn't find any information.
- [X] I already read and followed all the tutorials in the docs and didn't find an answer.
- [X] I already checked if it is not related to Typer but to [Click](https://github.com/pallets/click).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
import typer

app = typer.Typer()
@app.command()
def main(string:str, integer:int):
    print(f"{string}-{integer}")

app()
```

### Description

* Run nuitka using the args (--standalone --onefile code.py)
* Run the output
* Expected: Work normally
* Errors with `TypeError: readonly attribute`
Traceback:
```
│ in <module>:8 (my script)
│ in __call__:309 (typer)
│ in get_command:340 (typer)
│ in get_install_completion_arguments:107 (typer)
│ in get_click_param:909 (typer)
│ in get_param_callback:1007 (typer)
│ in update_wrapper:56 (functools)
```
### Operating System

Linux

### Operating System Details

Uname -A:
`Linux PC-01 6.6.30-2-MANJARO #1 SMP PREEMPT_DYNAMIC Wed May  8 17:46:43 UTC 2024 x86_64 GNU/Linux`

### Typer Version

0.12.3

### Python Version

3.12.3

**Comment by whiteath** [comment]
I encountered the same issue, and then I used Python 3.10, and the issue disappeared. Maybe you can try it.
